Source/FluentSpreadsheets.ClosedXML/Visitors/ClosedXmlVisitor.cs

```diff
@@ -14,53 +14,48 @@ public ClosedXmlVisitor(IXLWorksheet worksheet, Index index, Style style = defau
         _worksheet = worksheet;
     }
 
-    protected override Task StyleRangeAsync(Style style, IndexRange range, CancellationToken cancellationToken)
+    public override Task ApplyChangesAsync(CancellationToken cancellationToken = default)
+        => Task.CompletedTask;
```

Comment:
I thought ApplyChangesAsync will call `workbook.SaveAs` or `workbook.Save`, like GoogleSheetVisitor's ApplyChangesAsync, that will send google api requests. I think it's kinda misguiding that you're Applying Changes, but nothing's happening and you have to manually save workbook afterwards